packages/client/components/Dashboard.tsx

```diff
@@ -169,6 +173,10 @@ const Dashboard = (props: Props) => {
           </Switch>
           <Switch>
             <Route path='/new-meeting/:teamId?' component={NewMeetingRoot} />
+            <Route
```

Comment:
-1 I think we should move this one up to the `PrivateRoute` level as the "modal" is shown at that level (i.e. also on top of meetings). Basically add a second `<Switch` there like here and modify the top one to also consider the `backgroundLocation`. This way we would also render it on top of meetings.